backend/parser.py

```diff
@@ -62,7 +64,12 @@ def add_point(self, timestamp, datapoints: list, latency=-1):
         Returns: None
         """
 
-        self.timestamps.append(timestamp)
+        # Normalize timestamps to 0. Also convert from ns to ms.
+        if not self.initial_timestamp:
+            self.initial_timestamp = timestamp
+            self.timestamps.append(0)
+        else:
+            self.timestamps.append(10**-6*(timestamp - self.initial_timestamp))
```

Comment:
Add spaces between the *, i.e. `10**-6 * (`

I think this would cause you to lose precision on the timestamp anyway. It would be better to keep it as ns.